app/controllers/rents_controller.rb

```diff
@@ -17,4 +21,8 @@ def create
   def create_rent_params
     params.require(:rent).permit(:user_id, :book_id, :start_date, :end_date, :created_at)
   end
+
+  def user_not_authorized
+    render json: { error: 'Unauthorized user' }
```

Comment:
```suggestion
    head(:unauthorized)
```

Reply:
This has to do with the response status, right?

Reply:
I personally prefer:

```
def user_not_authorized(error)
  render json: { error: error.message }, status: :unauthorized
end
```

Reply:
Yes! Instead of returning a response with 200 status code, and a message like as: "Unauthorized user" we can return an empty response, with unauthorized status code :wink:

Reply:
Rami's suggestion includes both. A unauthorized status code and a response message.